packages/contracts/src.ts/index.ts

```diff
@@ -885,6 +887,7 @@ export class BaseContract {
             this._checkRunningEvents(runningEvent);
         };
 
+        // TODO: those won't work with txHash. Refactor it to use hedera txId
         event.getTransaction = () => { return this.provider.getTransaction(log.transactionHash); }
```

Comment:
This TODO should be resolved

Reply:
This cannot be resolved yet, as the `logs` endpoint does not provide us with the `txId` we need.

Reply:
Can we then handle it like we do in the other PR for `queryLogs` until then? We should not be providing a `getTransaction` function that will throw

Reply:
I misunderstood the first comment. Resolving soon.